src/common/options/global.yaml.in

```diff
@@ -1880,6 +1880,15 @@ options:
   services:
   - mon
   with_legacy: true
+- name: mon_config_history_size
+  type: size
+  level: advanced
+  desc: Defines the number of config history entries to be kept in the Monitors' store 
```

Comment:
Thank you for the plural posessive!
And thanks to @dvanders for having alerted me to the fact that we haven't been trimming these.  I have a customer right now who I think is suffering from this very issue, most likely led by the preponderance of `osd_memory_target` entries

If I understand the code correctly, I think we would do well to clarify a bit, something like:

>desc: The number of config history changes to be retained for each central configuration option.  This includes manual changes via 'ceph config' commands and adjustments to 'osd_memory_target' by the autotuner.